bugbug/similarity.py

```diff
@@ -207,6 +207,9 @@ def get_similar_bugs(self, query, k=10):
         # Get IDs of the k most similar bugs
         return [self.corpus[j[0]][0] for j in sims[:k]]
 
+    def get_distance(self, query1, query2):
+        pass
```

Comment:
Raise a NotImplemented exception instead of "pass", to specify that the function is not implemented

Reply:
https://docs.python.org/3.7/library/exceptions.html#NotImplementedError